adapters/handlers/grpc/v1/batch_delete.go

```diff
@@ -50,7 +50,7 @@ func batchDeleteParamsFromProto(req *pb.BatchDeleteRequest, scheme schema.Schema
 		return objects.BatchDeleteParams{}, err
 	}
 	filter := &filters.LocalFilter{Root: &clause}
-	if err := filters.ValidateFilters(scheme, filter); err != nil {
+	if err := filters.ValidateFilters(&scheme, filter); err != nil {
```

Comment:
it's not about efficiency, that is because the interface is implemented by the pointer receiver https://github.com/weaviate/weaviate/pull/4016/files#diff-c549ae044794cd678867a8b0ca6cf36dd056c918d6a68017ba76e8f90486e7d0R27

Reply:
The reason I mentioned this is because the current implementation already works with pass by value. However, pass by pointer would also make sense if you are using an interface